adm/style/acp_phpbb_ideas.html

```diff
@@ -28,16 +28,17 @@
 			<dd>{options.CONTENT}</dd>
 			<!-- IF options.KEY == 'ideas_poster_id' --><dd>[ <a href="{U_FIND_USERNAME}" onclick="find_username(this.href); return false;">{L_FIND_USERNAME}</a> ]</dd><!-- ENDIF -->
 		</dl>
+		<!-- IF options.KEY == 'ideas_base_url' -->
```

Comment:
The ideas forum setup button is a part of the options loop too, so we can't get rid of the condition. Otherwise we'll get it back as it was before.